code/__DEFINES/role_preferences.dm

```diff
@@ -37,7 +37,6 @@
 #define ROLE_LAVALAND			"Lavaland"
 #define ROLE_INTERNAL_AFFAIRS	"Internal Affairs Agent"
 #define ROLE_GANG				"Gangster"
-#define ROLE_SPECIAL			"Special Antagonists"
```

Comment:
I don't see why you need to remove the defines and bits that use them themselves. Mebbe just change the names and remove the preference bits

Reply:
This is never used in the code, and antags made with this system should use their own defines